### ISSUE TITLE
Add Automatic Module Names

### ISSUE THREAD

**Issue by KrishnaST** [softindex/datakernel#11]
The duplicate package names should not be used in multiple modules. Add Automatic Module Names for better integration with java 9+ modular applications.Make it future proof.

**Comment by KrishnaST** [comment]
![image](https://user-images.githubusercontent.com/9427954/64281329-89669900-cf70-11e9-9b66-782ddc47e6be.png)

**Comment by Fly-Style** [comment]
Hello, @KrishnaST. 
Could you share your **pom.xml** and **module-info.java** please?

**Comment by KrishnaST** [comment]
```
        <properties>
		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
		<maven.compiler.source>12</maven.compiler.source>
		<maven.compiler.target>12</maven.compiler.target>
	</properties>

	<dependencies>

		<dependency>
			<groupId>io.datakernel</groupId>
			<artifactId>http-launchers</artifactId>
			<version>3.0.0-beta1</version>
		</dependency>

		<dependency>
			<groupId>ch.qos.logback</groupId>
			<artifactId>logback-classic</artifactId>
			<version>1.2.3</version>
		</dependency>

	</dependencies>
```


```
module datakernel {

	exports paytabs;

	requires datakernel.common;
	requires datakernel.di;
	requires datakernel.http;
	requires datakernel.jmxapi;
	requires datakernel.launcher;
	requires datakernel.promise;
	requires http.launchers;
}
```

**Comment by KrishnaST** [comment]
Any Updates?

**Comment by dvolvach** [comment]
@KrishnaST , there is a fix in master branch, it should resolve this issue 
(several packages had to be renamed to prevent conflicts with automatic modules)
Execute ./install.sh to install it in your local Maven repository, then use 3.0.0-SNAPSHOT version in your project.

If everything is ok, it will be released soon to Maven central repository as 3.0.0 beta2 and release candidate.

**Comment by KrishnaST** [comment]
> 
> 
> @KrishnaST , there is a fix in master branch, it should resolve this issue
> (several packages had to be renamed to prevent conflicts with automatic modules)
> Execute ./install.sh to install it in your local Maven repository, then use 3.0.0-SNAPSHOT version in your project.
> 
> If everything is ok, it will be released soon to Maven central repository as 3.0.0 beta2 and release candidate.

couln't find the repository for 3.0.0-SNAPSHOT.
		```
<dependency>
			<groupId>io.datakernel</groupId>
			<artifactId>http-launchers</artifactId>
			<version>3.0.0-SNAPSHOT</version>
		</dependency>
```

**Comment by dvolvach** [comment]
@KrishnaST It's still in master branch of GitHub repository.

- Please clone git@github.com:softindex/datakernel.git
- Build and install it locally using ./install.sh script
- Now your projects should see 3.0.0-SNAPSHOT version

Also please note that Automatic Names have been added according to your advice, and names of Java Jigsaw modules are now different:
datakernel.eventloop -> io.datakernel.eventloop
datakernel.http -> io.datakernel.http
datakernel.jmxapi -> io.datakernel.jmx.api
http.launchers -> io.datakernel.launchers.http

Please let me know if you have any questions!

**Comment by dvolvach** [comment]
@KrishnaST The fix is now available in 3.0.0-beta2 public release, available on Maven central repository.

**Comment by KrishnaST** [comment]
Fixed with 3.0.0-beta2 . This framework has great future.
